client/src/components/Profile/index.tsx

```diff
@@ -107,13 +114,13 @@ export default class Profile extends React.Component<IProfileProps> {
                     </p>
                     <div className='d-flex followers justify-content-between'>
                         <div>
-                            <a href='#/' className='mr-2'><b>{posts}</b> posts</a>
+                            <button className='mr-2 but'><b>{posts}</b> posts</button>
```

Comment:
Give more clear class. Class but is not clear name

Reply:
fixed